src/app/index.ts

```diff
@@ -274,7 +275,11 @@ module.exports = yo.extend({
         // Copy project template files from project repository (currently only custom functions has its own separate repo)
         if (projectRepoBranchInfo.repo)
         {
-          git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', (projectRepoBranchInfo.branch) ? projectRepoBranchInfo.branch : 'master'], (err) => {
+          git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', (projectRepoBranchInfo.branch) ? projectRepoBranchInfo.branch : 'master'], (err) => {        
+            // //modify manifest guid and DisplayName
+            officeAddinManifest.modifyManifestFile(`${this.destinationPath()}/manifest.xml`, 'random', `${this.project.name}`);

```

Comment:
There would be an opportunity here to do the modifyManifestFile in parallel with the .git folder delete. Perhaps the file write time is not that significant, and likely dwarfed by the time to delete the .git folder, but just mentioned that they could be made to be done in parallel.

Reply:
@TCourtneyOwen You need the callback to be async and await officeAddinManifest.modifyManifestFile()  or use officeAddinManifest.modifyManifestFile().then(() => {
    //rest of the code
}